packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -289,6 +293,13 @@ class SearchAnchor extends StatefulWidget {
   /// {@macro flutter.widgets.editableText.textCapitalization}
   final TextCapitalization? textCapitalization;
 
+  /// Invoked upon user input on the search view.
+  final ValueChanged<String>? viewOnChanged;
+
+  /// Called when the user indicates that they are done editing the text in the
+  /// text field of a search view.
```

Comment:
You could another statement like "typically this is when the user presses the enter key" (assuming that's true).

Reply:
Updated!